apps/play/app/page-bk.tsx

```diff
@@ -21,7 +21,7 @@ export default function Home() {
 
   useEffect(() => {
     scrollRef?.current?.scrollIntoView({ behavior: "smooth" });
-  }, [scrollRef]);
+  }, []);
```

Comment:
`scrollRef` is a React component ref created with `useRef` hook thus it does not change between component re-renders. This means `scrollRef` being in a dependency array of a `useEffect` hooks has no effect.

Reply:
`scrollRef?.` the first optional chaining operator makes no sense here though, since `current` property always exist in a component ref — it just may be null sometimes. I'll change this in a separate commit.